docs/Versioning.md

```diff
@@ -26,16 +26,16 @@ For more details on ONNX Release versions, see [this page](https://github.com/on
 
 | ONNX Runtime release version | ONNX release version | ONNX opset version | ONNX ML opset version | Supported ONNX IR version | [WinML compatibility](https://docs.microsoft.com/en-us/windows/ai/windows-ml/)|
```

Comment:
What if we change the link from the general WinML docs page to this:
https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes

Reply:
ok